proto/cosmos/store/v2/commit_info.proto

```
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package cosmos.store.v2;
+
+import "gogoproto/gogo.proto";
```

buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)